lib/SlackBot.js

```diff
@@ -4,6 +4,7 @@ var express = require('express');
 var bodyParser = require('body-parser');
 var querystring = require('querystring');
 var async = require('async');
+var Cryptr = require('cryptr');
```

Comment:
The problem with this approach is `cryptr` uses AES as default. Due to AES, and it's algorithm if the database is compromised, yes the data can't be easily decrypted. But assume that a hacker has compromised your database instance, if that is the case, it's most likely that he/she may have access to your node instance also.